http_logs/challenges/default.json

```diff
@@ -136,6 +136,20 @@
           "warmup-iterations": 200,
           "iterations": 100,
           "target-throughput": 2
+        },
+        {
+          "name": "desc-sort-with-after-timestamp-after-force-merge-1-seg",
+          "operation": "desc_sort_with_after_timestamp",
+          "warmup-iterations": 200,
```

Comment:
these warmups and iterations are too big imo. 5 to 10 should be enough to have stable results ?

Reply:
addressed in b9f095f. But also wanted to confirm with @ebadyano that we are ok with 10 `warmup-iterations`